qa/suites/crimson-rados/basic/tasks/rados_python.yaml

```diff
@@ -12,4 +12,4 @@ tasks:
 - workunit:
     clients:
       client.0:
-        - rados/test_python.sh --eval-attr 'not (tier or snap or ec or bench or stats)'
+        - rados/test_python.sh --eval-attr 'not (wait or tier or snap or ec or bench or stats)'
```

Comment:
> wait_for_complete tests currently hanging with crimson-osd due to
> Objecter's (un-futurized) call to monc->get_version().

Why we would need to futurize `Objecter` or things it calls? Isn't that a client-side code?

Reply:
`Objecter` uses (e.g in `wait_for_latest_osdmap()`) Classic's MonClient get_version (`monc->get_version("osdmap", CB_Objecter_...`).
Crimson's MonClient get_version() is from `seastar::future<....> Client::get_version()` type. We should support Objecter's usage as well.

Reply:
Why? Do we have any potential user for it inside crimson-osd? Things external to crimson (e.g. tests, RBD; RADOS clients in general) should be fine with the classical `MonClient` IIUC.

Reply:
Updated the commit message after our discussion.